custom/panel_templates/Default/assets/css/sb-admin-2.min.css

```diff
@@ -7,4 +7,4 @@
  * Copyright 2011-2021 The Bootstrap Authors
```

Comment:
Just wondering what the changes to this file are for?

Reply:
~~well i just wasted my life~~
From: `font-family:'Font Awesome 5 Free'` to `font-family:'Font Awesome 6 Free'`

Reply:
Nice find, maybe that could be an override in custom.css instead?

Reply:
I'm happy for this to be "baked" into the minified CSS for now since that's probably how it is in the original.

Reply:
Downside is it will break if someone updates sbadmin like in #2624